### ISSUE TITLE
Installer Errors Bug Tracker

### ISSUE THREAD

**Issue by pulkit24** [pulkit24/ToME#238]
Hi,

Please use this tracker to describe your issues in detail if you receive the following errors:

1. ToME Setup fails and quits, showing an error message.
2. Nothing happens when double-clicking the ToME Setup.

Please tell me what you see (screenshot helps), your computer details, where is AoK installed, is it HD, did you have the previous ToME beta installed, are you launching as admin, and do you have other mods installed.

**Comment by Barhir** [comment]
![image](https://user-images.githubusercontent.com/42354006/44049445-0a9f8976-9f2c-11e8-93a2-92c9e29fdf38.png)
AoK is installed in the typical folder; it's not HD; I did have the previous ToME beta but I uninstalled it and then tried to install the new version; I tried launching as admin; I don't have other mods installed (except the 1.0c patch).
If I try ignoring the error of "AoETechTree.exe" (shown in the screenshot), when I try to launch it it shows an error message saying "AoETechTree.exe" is missing. Thanks in advance for your help!

**Comment by Barhir** [comment]
The problem is with the installer not having access in the directory. If there was a way to install to another destination, the desktop for example, I would then be able to manually move the files into "Age of Empires II". Hope you can fix this soon so I can try the new update out!

**Comment by pulkit24** [comment]
The installer requires other AoK files. It merges with the default graphics libraries and creates custom executables. It cannot work outside of an AoK install. 

What you can do though is copy the AoK folder entirely on to desktop, then install there. 

If the installer isn’t asking for install path, then you have a previous installation of ToME. Look for the uninstaller in your start menu and uninstall first. 

Let me know if this makes sense and if you can try any of this.

**Comment by Barhir** [comment]
Ok I have tried that, but now my antivirus software flags AoETechTree.exe as “Heur/APC (Cloud)” which is some sort of virus. Are you 100% sure this file is safe??? It has quarantined the file so when I try to launch the game it says that AoETechTree.exe is missing. Thanks for your help!

**Comment by pulkit24** [comment]
It is 100% safe. It makes sense why the antivirus is concerned, because this file injects the new ToME Tech Tree into the main game executable. This is the only way to get the complicated ToME Tech Tree loaded in the game (original AoE tech tree doesn't work with half of ToME's new features). 

Going ahead I'm working on a way to patch the executable so it won't need to inject the Tech Tree every time you launch the game. For now though, you need to put a rule in your antivirus to ignore this file.

**Comment by Barhir** [comment]
Ok I reinstalled everything including the c-patch and installed the new ToME. Then it didn't recognise the CD. After a bit of researching it turns out that the Collectors Edition (the edition I was installing from) already has the c-patch so when I installed the c-patch on top of it it creates a bug saying I need to insert the CD.
I'm so glad I've got that fixed now, thank you so much for your help! Now I need to try out this new version :)
